src/main/java/com/everis/entity/CarDto.java

```diff
@@ -0,0 +1,69 @@
+package com.everis.entity;
+
+import java.util.Date;
+
+
+public class CarDto {
+	
+	private String id;
+	private String brand;
+	private String country;
+	private Date createdAt;
+	private Date lastUpdated;
+    private Date registration;
+	public String getId() {
+		return id;
+	}
+	public void setId(String id) {
+		this.id = id;
+	}
+	public String getBrand() {
+		return brand;
+	}
+	public void setBrand(String brand) {
+		this.brand = brand;
+	}
+	public String getCountry() {
+		return country;
+	}
+	public void setCountry(String country) {
+		this.country = country;
+	}
+	public Date getCreatedAt() {
+		return createdAt;
+	}
+	public void setCreatedAt(Date createdAt) {
+		this.createdAt = createdAt;
+	}
+	public Date getLastUpdated() {
+		return lastUpdated;
+	}
+	public void setLastUpdated(Date lastUpdated) {
+		this.lastUpdated = lastUpdated;
+	}
+	public Date getRegistration() {
+		return registration;
+	}
+	public void setRegistration(Date registration) {
+		this.registration = registration;
+	}
+
+	public CarDto(String id, String brand, String country, Date createdAt, Date lastUpdated, Date registration) {
```

Comment:
Well done, you've encapsulated the mapping code here, in a future if this needs to changes there is only a single place to do it.
This could be improved, some weak points:
- The constructor attributes are too open, for example id and brand are string types, what if the user of this constructor accidentally swap the values ?
- Manually mapping the values is a lot of boilerplate code and every time a new field is added we must update this part of the code, the constructor ( and all the places where the constructor is used ) and the mapping. There are libraries that does this kind of mapping for us : 
-- http://modelmapper.org/
-- https://mapstruct.org/
-- http://commons.apache.org/proper/commons-beanutils/

Please have a look into these libraries and think if it is possible to improve this mapping logic. Do not waste too much time in it.